exporter/prometheus/prometheus.go

```diff
@@ -51,23 +50,9 @@ type Options struct {
 	OnError   func(err error)
 }
 
-var (
-	newExporterOnce      sync.Once
-	errSingletonExporter = errors.New("expecting only one exporter per instance")
-)
-
 // NewExporter returns an exporter that exports stats to Prometheus.
-// Only one exporter should exist per instance
+// Most users should only register a single Prometheus exporter per process.
```

Comment:
I'd not add this to the docs.

This is what the Prometheus community is already aware about, don't document it. Document things if there is something not expected.